app/services/catalog/create_approval_request.rb

```diff
@@ -46,12 +46,11 @@ def request_body_from(order_item)
     end
 
     def workflows(portfolio_item)
-      portfolio_item.resolved_workflow_refs
+      []
```

Comment:
This method will be totally removed. Placed temporarily until https://github.com/ManageIQ/catalog-api/pull/462 is merged so more specs pass.